dev/set_matrix.py

```diff
@@ -199,7 +226,7 @@ def filter_versions(versions, min_ver, max_ver, excludes=None):
     assert all(v in versions for v in excludes)
 
     versions = {v: t for v, t in versions.items() if v not in excludes}
-    versions = {v: t for v, t in versions.items() if is_final_release(v)}
+    versions = {v: t for v, t in versions.items() if is_final_release(v) or is_post_release(v)}
```

Comment:
Not sure if this is the right usage for a post release, but XGBoost released 1.3.0.post0 right after they relreased 1.3.0 which contains a bug: https://pypi.org/project/xgboost/#history

Reply:
scikit-learn also has some post releases: https://pypi.org/project/scikit-learn/#history